department-of-veterans-affairs/vets-website
# VAGOV-2855: Display 'Other VA locations' via Facility ID

## Description
Displays "Other VA Locations" on the locations page of a facility

I decided to add another react widget because of how often the designs change - I want the flexibility from the get-go since the design is in flux.

## Testing done
Locally with staging and dev data
Pre-req: May have to make sure there is data in the Pittsburgh Health Care system for "Other VA Locations" to populate `fieldOtherVaLocations` http://staging.cms.va.gov/node/318/edit?destination=/admin/content

You can fill in `nca_120` for the soldier lot that is in the mock ups

1. re-build site with either staging or dev data
2. browse to http://localhost:3001/pittsburgh-health-care/locations/
3. scroll all the way down to see the "Other VA Locations" section

## Screenshots
![localhost_3001_pittsburgh-health-care_locations_ (2)](https://user-images.githubusercontent.com/19178435/58582737-52614880-8207-11e9-9501-69df5de7423e.png)

## Acceptance criteria
https://va-gov.atlassian.net/browse/VAGOV-2855

## Definition of done
